QUEST_LV_0100.tsv

```diff
@@ -2259,7 +2259,7 @@ QUEST_LV_0100_20150317_002254	I'm furious and upset. {nl} If we got here earlier
 QUEST_LV_0100_20150317_002255	$Dear Revelator, I have a favor to ask of you. {nl}Can you recover my comrades' mementos?
 QUEST_LV_0100_20150317_002256	{memo X}It's bad enough that they were killed by the Vubbes, but even their keepsakes are being abused. How awful...
 QUEST_LV_0100_20150317_002257	{memo X}Thank you. {nl} My comrades will now be able to rest in peace.
-QUEST_LV_0100_20150317_002258	$Oh my God. The Vubbes stole all our food supplies. {nl} They even took our food for today... Please help us, we need your strength.
+QUEST_LV_0100_20150317_002258	$Oh my god. The Vubbes stole all our food supplies. {nl} They even took our food for today... Please help us, we need your strength.
```

Comment:
Do you really refer to God when you say this xD?
If not really, then it's just a figurative god right?